join.html

```diff
@@ -11,16 +11,16 @@
   <h3>Meet Our Current Team</h3>
 
   <div class="usa-width-one-third">
-    <img src="{{ site.baseurl }}/assets/site/img/headshot.jpg" />
-    <h3>Crystal</h3>
+    <img src="{{ site.baseurl }}/assets/site/img/krystal.jpg" height=100% width=100% />
+    <h3>Krystal</h3>
     <h4>Data Scientist</h4>
-    <p>I'm happiest in R REPL, slicing and dicing through data mountains.</p>
+    <p><q>The opportunity to solve unique data problems for any federal agency, not just the Department of Commerce, was why I joined NTIS as a Data Scientist. Previously, I primarily focused on weather data and I am excited for the challenge of working with data outside of my subject matter expertise.  Teaming with the private sector, specifically our new Joint Venture Partners, will allow for innovation, collaboration, and the ability to stay involved with up and coming technologies.</q></p>
   </div>
   <div class="usa-width-one-third">
     <img src="{{ site.baseurl }}/assets/site/img/headshot.jpg" />
-    <h3>Paul</h3>
-    <h4>Data Scientist</h4>
-    <p>I love working on the infrastructure and platforms to host data.</p>
+    <h3>Derssie</h3>
+    <h4>Chief Data Scientist</h4>
+    <p>Derssie Mebratu received the B.S. from McMaster university and M.Eng and Ph.D. degree in electrical engineering from Howard University. Dr. Mebratu’s research interests include Statistical signal processing, Predictive analytics, Bayesian statistical analysis, Linear dynamic model, Clustering algorithm, Queuing theory, Neural network, Semantics ontology and Text analytics. Prior to coming to NTIS, Dr. Mebratu was working as Algorithm Scientist at Orbis Technologies and as  research engineering at NASA Goddard.  Dr. Mebratu is currently working as a Chief Data Scientist at the National Technical Information Service in the Department of Commerce, to develop a cutting-edge technology in the field of Data science.</p>
```

Comment:
I think a shorter paragraph will do much better here for Derssie. I'd drop the educational pedigree. While it's important, it's less useful in encouraging others to apply.